README.md

```diff
@@ -29,6 +29,18 @@ Plug 'nvim-tree/nvim-web-devicons'             " Optional
 call plug#end()
 ```
 
+### lazy.nvim
+
+```lua
+return {
+    "seblj/nvim-tabline",
+    requires = { "nvim-tree/nvim-web-devicons" }, -- Optional
```

Comment:
Isn't this `dependencies` and not`requires`? I think that is packer.nvim

Reply:
Yes, absolutely, don't know how I messed up this copy-paste.